tests/performance_tests/performance_test_basic.py

```diff
@@ -493,16 +492,13 @@ def runTest(self) -> bool:
                 self.testHelperConfig._killWallet,
                 self.testHelperConfig.keepLogs,
                 self.testHelperConfig.killAll,
-                self.testHelperConfig.dumpErrorDetails
+                self.testHelperConfig.dumpErrorDetails,
+                trxGenLauncher=self.cluster.trxGenLauncher
```

Comment:
`TestHelper` already knows about `cluster`, and now that you've switched over to using the `self.cluster.trxGenLauncher` instance, it would seem to follow the pattern here to add a `self.testHelperConfig.killAllTrxGen` flag instead of passing in the `trxGenLauncher` itself.

Although, now that I'm looking at it more closely, is adding this to `TestHelper.shutdown` even necessary since `cluster.killall` will clean up the `self.trxGenLauncher` if the `self.testHelperConfig.killAll` flag is set?

Reply:
This appears correct. Now that it is set in the cluster, it should be handled since we are using killall.